docs/navigation-parameters-reference.md

```diff
@@ -843,6 +843,15 @@ The global search is an input field available in the top navigation bar. The sea
 - **description**: disables the on:keyUp and other internal handlers on the search input field. It is a plain input field then, which can be used to attach your own handlers. If set to `true`, a **searchProvider** must be defined in order to show the search field, which can contain your custom logic. It is recommended to initialize your custom logic in the [**lifeCycle.luigiAfterInit**](lifecycle-hooks.md#luigiafterinit) hook.
 - **since**: 1.5.0
 
+### searchFieldCentered
+- **type**: boolean
+- **description**: The search input field will be rendered in the center of the shellbar.
+- **since**: NEXTRELEASE
+
+### globalSearchCenteredCancelButton
+- **type**: string
+- **description**: defines the label of the cancel button. It will be displayed if you want to hide the search field on a smaller viewport.
+- **since**: NEXTRELEASE
```

Comment:
Ordering the parameters in alphabetical order according to the new convention
```suggestion
### globalSearchCenteredCancelButton
- **type**: string
- **description**: defines the label of the cancel button. It will be displayed if you want to hide the search field on a smaller viewport.
- **since**: NEXTRELEASE

### searchFieldCentered
- **type**: boolean
- **description**: The search input field will be rendered in the center of the shellbar.
- **since**: NEXTRELEASE

```

Reply:
Good point Alex 👍  :). Totally forgot